compiler/dalgona/analysis/AnalysisTemplate.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2022 Samsung Electronics Co., Ltd. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""
+A template for anlaysis code.
+This template shows how to access the information of each operator inside hooks.
+Users can write their own hooks by modifying this file.
```

Comment:
I assume this file will be installed... inside `/usr/share/one/bin/...`
That is will need sudo to edit.
Q) Where is this file going to be installed?

Reply:
Ah, maybe "Users can write their own hooks by copy and modifing this file" ?

Reply:
This file is like a document (manual), not an executed code. So it will not be installed in the `bin` directory.

Only one executable `dalgona` will be installed in `/usr/share/one/bin/`.